data.sql

```diff
@@ -33,4 +34,61 @@ INSERT INTO animals (
   11,
   TRUE,
   5
+);
+
+-- DAY 2
+INSERT INTO animals (
+  name,
+	date_of_birth,
+	weight_kg,
+	neutered,
+	escape_attempts
+) VALUES (
+  'Charmander',
+  '2020-02-08'::date,
+  -11,
+  TRUE,
+  0
+),
+(
+  'Plantmon',
+  '2021-11-15'::date,
+  -5.7,
+  TRUE,
+  2
+),
+(
+  'Squirtle',
+  '1993-04-02'::date,
+  -12.13,
+  FALSE,
+  3
+),
+(
+  'Angemon',
+  '2005-06-12'::date,
+  -45,
+  TRUE,
+  1
+),
+(
+  'Boarmon',
+  '2005-06-12'::date,
+  20.4,
+  TRUE,
+  7
+),
```

Comment:
Great job @jlvFlores 🤝

- [ ] Kindly ensure you entered the correct date for `Boarmon`.